src/NzbDrone.Core/Indexers/RssParser.cs

```diff
@@ -120,7 +120,8 @@ protected virtual ReleaseInfo CreateNewReleaseInfo()
 
         protected virtual bool PreProcess(IndexerResponse indexerResponse)
         {
-            if (indexerResponse.HttpResponse.StatusCode != HttpStatusCode.OK)
+            // Server Down HTTP Errors are handled in HTTPIndexerBase so ignore them here
+            if (indexerResponse.HttpResponse.StatusCode != HttpStatusCode.OK && !indexerResponse.HttpResponse.IsServerNotAvailable)
```

Comment:
Don't we still want to throw if we got an unexpected response instead of trying to process responses?

Reply:
they were getting thrown in httpindexerbase when testing in downstream; should be same here

Reply:
Right, but if they're getting thrown and still reaching here then we shouldn't try to process the response.

Reply:
@mynameisbogdan any insight here

quick testing has this concept working fine in prowlarr but doesn't really work in sonarr (assume same for downstream starr)

Reply:
little lost in the weeds for where exactly to apply that